tests/contrib/pytest/test_pytest.py

```diff
@@ -1228,11 +1234,18 @@ def test_ok(self):
         rec = self.inline_run("--ddtrace", file_name)
         rec.assertoutcome(failed=1, passed=1)
         spans = self.pop_spans()
-        test_span_skipped = spans[0]
-        test_span_ok = spans[1]
-        test_suite_span = spans[4]
-        test_session_span = spans[2]
-        test_module_span = spans[3]
+        if len(spans) >= 7:
```

Comment:
The use of number literals in this code makes it hard to change. You might consider finding a way to abstract these different cases into a function or class that uses descriptive naming to clarify the separation of concerns between these two paths.

Reply:
Good point, I just reverted `test_pytest.py` as `unittest` won't run if `pytest` is enabled (we support `unittest` with our `pytest` integration anyway)